README.md

```diff
@@ -411,15 +444,12 @@ The parameters that can be passed into the `hmrcStandardPage` are as follows:
       | `backLink`                                 | Passing a value will display a back link                          | `Some(BackLink(href = ..., attributes = ...))`              |
       | `cspNonce`                                 | This will be bound to hmrcHead, hmrcScripts and govukTemplate     | CSPNonce.get                                                |
 
-### FullWidthPageLayout should only be used by internal services
 
-The default fixed width layout is for public services.
+### Creating consistent page headings
```

Comment:
Could we add in this with the restructuring, got a bit of tech debt to properly deprecate the helpers but if we're changing the documentation I think it makes sense to drop this in at the same time

```suggestion
### Creating consistent page headings

> **Warning**
> The [hmrc guidance for creating headings with a section (caption)](https://design.tax.service.gov.uk/hmrc-design-patterns/page-heading/) has recently changed. The following helpers are still available but this is no longer the recommended approach. Consult the linked documentation for examples of the new recommendation.
```